src/chocolatey.resources/helpers/functions/Get-ChocolateyWebFile.ps1

```diff
@@ -17,7 +17,7 @@
 function Get-ChocolateyWebFile {
 <#
 .SYNOPSIS
-Downloads a file from the internets.
+Downloads a file from the internet.

```

Comment:
Some projects are cute, this one doesn't _appear_ to be in that family, and thus I don't think the `s` is worth keeping. But, I could be wrong.

Reply:
This was actually meant to be funny. That said, better to have the change.